### PR TITLE
Add wait_template for 0.39

### PR DESCRIPTION
**Pull request by pvizeli**
**Description:**

Add description for `wait_template` in script

### CODE REVIEW

File: source/getting-started/scripts.markdown [hunk 1]
```diff
@@ -74,6 +74,25 @@ delay:
...
+
+```yaml
+# wait until a valve is < 10 or abort after 1 minutes.
+wait_template: "{{ states.climate.kitchen.attributes.valve < 10 }}"
+timeout: 00:10:00
+```
+
+```yaml
+# alternative
+wait_template: "{% if"
```
Comment (arsaboo) [inline source/getting-started/scripts.markdown]: This is incomplete...

File: source/getting-started/scripts.markdown [hunk 2]
```diff
@@ -74,6 +74,25 @@ delay:
...
+
+```yaml
+# wait until media player have stop the playing
+wait_template: "{{ states.media_player.floor.states == 'stop' }}"
+```
+
+```yaml
+# wait until a valve is < 10 or abort after 1 minutes.
+wait_template: "{{ states.climate.kitchen.attributes.valve < 10 }}"
+timeout: 00:10:00
```
Comment (arsaboo) [inline source/getting-started/scripts.markdown]: Shouldn't it be `timeout: 00:01:00` for aborting after 1 minute?

File: source/getting-started/scripts.markdown [hunk 3]
```diff
@@ -74,6 +74,25 @@ delay:
 # Valid formats include HH:MM and HH:MM:SS
 delay: {% raw %}'00:{{ states.input_slider.minute_delay.state | int }}:00'{% endraw %}
 ```
+### {% linkable_title Wait %}
+
+Wait until some things are complete. We support at the moment `wait_template` for waiting until a condition is `true`, see also on [Template-Trigger](/getting-started/automation-trigger/#template-trigger). It is possible to set a timeout after that will the script abort his execution. Timeout have same syntax as `delay`.
```
Comment (arsaboo) [inline source/getting-started/scripts.markdown]: Unlike `delay` where you need to specify the wait time, `wait` uses a template to determine the wait time. You can pause the execution of the script until the condition is satisfied. At the moment, we support `wait_template` for waiting until a template condition is `true`, see also on [Template-Trigger](/getting-started/automation-trigger/#template-trigger). It is possible to set a timeout after which the script will abort execution. Timeout have the same syntax as `delay`.

(quick question - will `timeout` abort the entire script as suggested above or move to the next instruction)

Comment (pvizeli) [inline source/getting-started/scripts.markdown]: Yeah it abort the hole script.

**Pull request merged**